app/src/main/java/fr/free/nrw/commons/Media.kt

```diff
@@ -114,6 +114,30 @@ class Media constructor(
             else
                 ""
 
+    /**
+     * Gets media display title
+     * @return Media title
+     */
+
+    fun getDisplayAuthor(): String{
+        if (author == null)
+            return ""
+
+        val completeHtmlTagCheck = "<.*>(.*)<.*>".toRegex() // Detect if complete html tags present
+        var completeCheckResult = completeHtmlTagCheck.findAll(author!!) // Find matching cases, return text between tag
+        if(completeCheckResult.count()==1) return completeCheckResult.toList().get(0).groupValues.get(1) // Get matching result and return text between tag - text between tag is inside a capture group
```

Comment:
Please use these spacing conventions: https://kotlinlang.org/docs/coding-conventions.html#horizontal-whitespace

Also, better split this into 3 lines:
```
// comment
if ...
    return ...

```
with a new line before and after.